### PR TITLE
Jhh prefix

### PR DESCRIPTION
**Pull request by ray-ray**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/tests/integration/conftest.py [hunk 1]
```diff
@@ -9,4 +9,4 @@ def domain():
     """
     :return: Domain of the running service.
     """
-    return 'http://0.0.0.0:5000'
+    return 'http://0.0.0.0:{{cookiecutter.dc_port}}'
```
Comment (imjohsep) [inline {{cookiecutter.project_slug}}/tests/integration/conftest.py]: Do you think we should use port 5000 and require npgateway to be running in order to execute integration tests?

Comment (ray-ray) [reply {{cookiecutter.project_slug}}/tests/integration/conftest.py]: My preference is to be able to hit the server directly while running npgateway, so I need the port.

If we always go through npgateway, then it means you can't test unexposed routes (like /healthz), and it adds another moving part to the test.

Comment (imjohsep) [reply {{cookiecutter.project_slug}}/tests/integration/conftest.py]: I ran into /healthz failing integration tests today for that exact reason.

If we intend on running integration tests against the k8 environments we will need to use npgateway or create an ingress for each service. In both cases we will need a config to toggle the base route. 

Maybe we should revisit the plans for integration tests. Particularly, when and where do we want to run them.

Comment (ray-ray) [reply {{cookiecutter.project_slug}}/tests/integration/conftest.py]: A few thoughts here:

1. For debug/testing purposes in non-live environments, I think it's useful to add an ingress for every service.

2. The route in conftest.py would have to be updated based on the test environment regardless of whether we're going through npgateway. I do this manually now but plan to handle this in an automated (probably ENV var) way once we have our test environments better established.

3. I could also see an argument that different sets of integration tests should run in different environments. In our locals, we run all of them, but perhaps we only run tests on exposed routes in k8 envs and perhaps we only run a subset of sanity/smoke tests against prod.

4. Maybe we should promote some (or write new) tests to full-fledged system tests where the expectation is that they will run through npgateway and treat the entire system as a black box.

**Pull request merged**